naas/models/subscriber.py

```diff
@@ -29,17 +31,19 @@ def full_name(self):
         """Returns the subscriber full_name"""
         return f'{self.first_name()} {self.last_name()}'
 
-    def email(self):
-        """Returns the subscriber email"""
-        return self.attributes.get('email')
+    def email_address(self):
+        """Returns the subscriber email address"""
+        return self.attributes.get('email_address')
 
     def created_at(self):
         """Returns the created at timestamp value"""
-        return self.attributes.get('created_at')
+        return datetime.datetime.strptime(
+            self.attributes.get('created_at'), '%Y-%m-%dT%H:%M:%S%z')
 
     def updated_at(self):
         """Returns the updated at timestamp value"""
-        return self.attributes.get('updated_at')
+        return datetime.datetime.strptime(
+            self.attributes.get('updated_at'), '%Y-%m-%dT%H:%M:%S%z')
 
```

Comment:
@arthuroe - Almost all of the models will include these timestamps (`created_at`, `updated_at`), so anytime we have a `Date`, `Time`, or `DateTime` object we will want to coerce it properly. This way the consumer can use date/time libraries to work with and format the dates.

This is helpful to:

* Format the dates as they choose
* At the collection level, the client can implement ranges of 'find all records between date X and Y'